classes/File.js

```diff
@@ -14,11 +15,7 @@ class File {
     this.accessToken = accessToken
     this.siteName = siteName
     this.baseEndpoint = null
-    this.branchRef = 'staging'
-  }
-
-  setBranchRef(branchRef) {
-    this.branchRef = branchRef
+    this.branchRef = BRANCH_REF
```

Comment:
I don’t have strong feelings about this but maybe we could just omit the use of this.branchRef and just assign the param “branch” to BRANCH_REF in all the calls

Reply:
Removed all mentions of `this.branchRef` and use `BRANCH_REF` instead